### DISCUSSION TITLE
Cheat sheet for Equipment Info

### DISCUSSION THREAD

**Discussion by Patrick-Station** [sdlpal/sdlpal#220] [Q&A]
Is there a cheat sheet for the equipment and what their stats are? Like an excel sheet perhaps?

**Comment by weimzh** [comment]
this can be found in the manual of the game (which is IIRC included in the Steam version as a pdf)

you may also find this useful :
https://sdlpal.github.io/wszw/item.htm

unfortunately both are Chinese only.
